packages/datagateway-common/src/detailsPanels/dls/visitDetailsPanel.component.tsx

```diff
@@ -295,6 +309,70 @@ const VisitDetailsPanel = (
           </StyledGrid>
         </div>
       )}
+      {investigationData.parameters && (
+        <div
+          id="investigation-parameters-panel"
+          aria-labelledby="investigation-parameters-tab"
+          role="tabpanel"
+          hidden={selectedTab !== 'parameters'}
+        >
+          <StyledGrid id="parameter-grid" container direction="column">
+            {investigationData.parameters.length > 0 ? (
+              investigationData.parameters.map((parameter) => {
+                if (parameter.type) {
+                  switch (parameter.type.valueType) {
+                    case 'STRING':
+                      return (
+                        <Grid key={parameter.id} item xs>
+                          <Typography variant="overline">
+                            {parameter.type.name}
+                          </Typography>
+                          <Typography>
+                            <b>{parameter.stringValue}</b>
+                          </Typography>
+                        </Grid>
+                      );
+                    case 'NUMERIC':
+                      return (
+                        <Grid key={parameter.id} item xs>
+                          <Typography variant="overline">
+                            {parameter.type.name}
+                          </Typography>
+                          <Typography>
+                            <b>{parameter.numericValue}</b>{' '}
+                            {parameter.type.units}
+                          </Typography>
+                        </Grid>
+                      );
+                    case 'DATE_AND_TIME':
+                      return (
+                        <Grid key={parameter.id} item xs>
+                          <Typography variant="overline">
+                            {parameter.type.name}
+                          </Typography>
+                          <Typography>
+                            <b>
+                              {parameter.dateTimeValue &&
+                                parameter.dateTimeValue.split(' ')[0]}
```

Comment:
If this is trying to get just the date - again it's probably preferred to use `date-fns` to be more resilient to future timestamp format changes.